xbrowse_server/base/models.py

```diff
@@ -321,10 +321,23 @@ def get_reference_population_slugs(self):
     def get_options_json(self):
         d = dict(project_id=self.project_id)
         d['id'] = self.id
-        d['reference_populations'] = (
-            [{'slug': s['slug'], 'name': s['name']} for s in settings.ANNOTATOR_REFERENCE_POPULATIONS] +
-            [{'slug': s.slug, 'name': s.name} for s in self.private_reference_populations.all()]
-        )
+        if self.has_elasticsearch_index():
+            d['db'] = "elasticsearch"
+            d['reference_populations'] = [
+                {"slug": "1kg_wgs_phase3", "name": "1000G v3"},
```

Comment:
should these also live in a constant somewhere?

Reply:
yes they should